curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153908a366afb4d57185c8d.md

```diff
@@ -7,16 +7,25 @@ dashedName: step-12
 
 # --description--
 
-Flexbox can be thought of as having two axes, the main axis and the cross axis. The main axis is determined by the `flex-direction` property. If `flex-direction` is set to `row` or `row-reverse`, the main axis is horizontal. If `flex-direction` is set to `column` or `column-reverse`, the main axis is vertical.
+Flexbox has a main and cross axis. The main axis is defined by the `flex-direction` property, which has four possible values:
 
-Give your `#gallery` selector a `flex-direction` property set to `row`.
+- `row` (default): horizontal axis with flex items from left to right
+- `row-reverse`: horizontal axis with flex items from right to left
+- `column`: vertical axis with flex items from top to bottom
+- `column-reverse`: vertical axis with flex items from bottom to top
+
+**Note**: The axes and directions will be different depending on the text direction. The values shown are for a left-to-right text direction.
+
+Try the different values to see how they affect the layout.
```

Comment:
```suggestion
```

Reply:
Why? I think it's worth encouraging the camper to try the values and see how they affect the layout.

Reply:
Some aren't in the field for learning. For them, if they can see something that can be skipped, and test it at later date, they would.

 I am afraid, if they "see try something out", they won't.

Reply:
> Some aren't in the field for learning.

I don't really know what you mean by this?

> I am afraid, if they "see try something out", they won't.

If they don't, they don't. I don't see how that is relevant to encouraging it.

Reply:
I'm with lasjorg on this - I'm a big fan of encouraging campers to experiment with things. It's a great way to learn.